index.html

```diff
@@ -37,7 +37,7 @@
         <!-- Navbar Starts -->
         <header class="header-menu" id="head_top">
             <div class="main-menu" style="margin-top: 9%;">
-                <nav class="navbar fixed-top navbar-expand-lg navbar-light " style="background-color: white;">
+                <nav class="navbar fixed-top navbar-expand-lg navbar-light pb-xl-0 pb-lg-0 pb-md-0 pb-sm-0 pb-0">
                     <a class="navbar-brand" href="#"><img class="img" src="./assets/mozguj-horizontal-black.png"
                             alt="user" height="100px" width="210px"></a>
```

Comment:
Avoid Height width in HTML itself!

Reply:
changes have done please do check it once.